src/etc/udev/rules.d/65-vyos-net.rules

```diff
@@ -0,0 +1,26 @@
+# These rules use vyos_net_name to persistently name network interfaces
+# per "hwid" association in the Vyatta configuration file.
```

Comment:
Maybe remove the reference to Vyatta here?

Reply:
Yes, thanks !